certified-connectors/N-able Cloud User Hub/readme.md

```diff
@@ -0,0 +1,58 @@
+## Cloud User Hub
+Cloud User Hub is a unified Microsoft CSP portal that lets you automate common tasks across all Microsoft 365 apps without switching consoles or customers.
+
```

Comment:
@VincentMejan please format the readme.md file according to [the template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md?plain=1). The publisher is missing. All of the required headings should be present.

Reply:
Fixed in latest commit